rpaas/k8s.go

```diff
@@ -795,6 +770,27 @@ func (m *k8sRpaasManager) eventsForPod(ctx context.Context, podName, ns string)
 	return eventList.Items, nil
 }
 
+func newSecretForCertificates(instance v1alpha1.RpaasInstance, data map[string][]byte) *corev1.Secret {
+	hash := fmt.Sprintf("%x", sha256.Sum256([]byte(fmt.Sprint(data))))
```

Comment:
Hashing `fmt.Sprint(data)` makes this code depends on the assumption that the order of keys in a printed map is always the same. This may be true for newer Go version but was not always the case, and in any way we would be depending on an implementation detail.
It's ugly but converting this map into a `[][]byte{key1, value1, key2, value2,...}` after sorting the keys would be a safer way to do this.

Reply:
Yep, make sense... I made a draft for this suggestion at https://play.golang.org/p/t28S6xiZpXT, can you review?